ci-operator/config/openshift/assisted-service/openshift-assisted-service-cloud_hotfix_releases.yaml

```diff
@@ -1,4 +1,8 @@
 base_images:
+  assisted-image-service:
```

Comment:
also requires a backport (although not a blocker in this case)

Reply:
You right, I removed it for now